staging/cse/windows/provisioningscripts/kubeproxystart.ps1

```diff
@@ -33,8 +70,23 @@ if ($KubeproxyFeatureGates -contains "WinDSR=true") {
     $global:KubeproxyArgList += @("--enable-dsr=true")
 }
 
-if ($KubeproxyFeatureGates.Count -ne 0) {
-    $global:KubeproxyArgList += @("--feature-gates=" + ($KubeproxyFeatureGates -join ","))
+$featureGateArgs = ""
+foreach ($feature in $KubeproxyFeatureGates) {
+    # IPv6DualStack feature gate should not be passed to kube-proxy in >= 1.25.0
+    # https://github.com/kubernetes/kubernetes/blob/ef70d260f3d036fc22b30538576bbf6b36329995/pkg/features/kube_features.go#L945
+    if (($feature -like "IPv6DualStack=*") -and ((Compare-SemanticVersion -Version1 $KubernetesVersion -Version2 "1.25.0") -ge 0)) {
+        continue
```

Comment:
Can we remove `featureGates["IPv6DualStack"] = true` in `GetKubeProxyFeatureGatesWindowsArguments` in pkg/agent/datamodel/types.go?

Reply:
ah actually that might be better. I was thinking we were checking the `IPv6DualStack=true` feature gate somewhere else but it looks like we are really using `IsIPv6DualStackFeatureEnabled` and then setting `$global:IsDualStackEnabled` based on that.

Reply:
For now I'm going to leave this bit in kubeproxystart.ps1 to remove it from the list in addition to removing from types.go. Since we need this change relatively quickly, I want to be able to use a new CSE version and will remove this in a follow up PR so that it is fully controlled by types.go